weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ChatView/MessagePanel.tsx

```diff
@@ -7,34 +8,98 @@ import {ToolCalls} from './ToolCalls';
 import {Message} from './types';
 
 type MessagePanelProps = {
+  index: number;
   message: Message;
   isStructuredOutput?: boolean;
+  isChoice?: boolean;
+  isNested?: boolean;
+  pendingToolResponseId?: string;
```

Comment:
This also appears to be unused

Reply:
isNested is use for margins on the toolCallResponses below

isChoice
pnedingToolResponseId
are used for the messagePanel buttons i can remove for this pr